### DISCUSSION TITLE
sui 浏览器调用合约 报错

### DISCUSSION THREAD

**Discussion by v1xingyue** [NonceGeek/Move-Camp#260] [Q&A]
添加了一个不需要coin 的合约，从浏览器调用报错: 

```move
    public entry fun buy_free_bread(
        ctx: &mut TxContext
    ) {
        transfer::transfer(Bread { id: object::new(ctx) }, tx_context::sender(ctx))
    }
```
错误如下: 

<img width="708" alt="image" src="https://user-images.githubusercontent.com/974169/215903996-fe0525e7-6835-4d40-91c3-a787b0f7a6c0.png">

**Comment by qiwihui** [comment]
这个报错看着是区块链RPC版本和前端sdk版本不匹配导致的数据结构不匹配，建议查一下这两个之间版本的区别。
如果用命令行可以调用成功吗？以及前端调用可否提供？
